sdks/python/apache_beam/runners/dataflow/dataflow_runner.py

```diff
@@ -473,10 +474,19 @@ def run_pipeline(self, pipeline, options):
 
     use_fnapi = apiclient._use_fnapi(options)
     from apache_beam.transforms import environments
-    self._default_environment = (
-        environments.DockerEnvironment.from_container_image(
-            apiclient.get_container_image_from_options(options),
-            artifacts=environments.python_sdk_dependencies(options)))
+    if options.view_as(DebugOptions).lookup_experiment(
+        'prebuild_sdk_container'):
```

Comment:
Should this be a top level option similar to `docker_registry_push_url` or vice versa? It probably sense to make either both options an experiment, or both as top level options.

Reply:
make sense, moved to experiment.

Reply:
+1 to being consistent. I would lean towards both being top-level options.

Reply:
done.